packages/slider-react/documentation/Example.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+import { ExampleComponentProps } from "../../../doc-utils";
+import { Slider } from "../src";
+import { LabelVariant } from "@fremtind/jkl-core";
+
+export const Example = ({ boolValues, choiceValues }: ExampleComponentProps) => {
+    const variant = choiceValues && (choiceValues["Etikettvariant"] as LabelVariant);
+    const errorLabel = boolValues && boolValues["Med feil"] ? "Du må velge en verdi" : undefined;
+    const helpLabel =
+        boolValues && boolValues["Med hjelpetekst"] ? "1 = veldig misførnøyd, 5 = veldig fornøyd" : undefined;
+
+    const [value, setValue] = useState<number>();
+
+    return (
+        <Slider
```

Comment:
I slider komponenten bør man kunne kanskje trykke på tallene også? Også synes jeg det klikkbare området på progressbaren er usynlig, kanskje det burde være noe synlig klikkbart inaktiv område, sånn at brukeren skjønner at man kan trykke på de områdene for å navigere seg dit

Reply:
Ja denne komponenten har forbedringspotensiale, men her er det muligens snakk om å måtte gå over til en mer innfløkt variant, siden det er litt begrenset hva man får til ved å bruke en standard `<input type="range" />`-tag. Vi har derfor funnet ut at denne går ut som den er, slik at vi får mulighet til å teste den og se hva som er de viktigste forbedringspunktene, og dermed hvilken løsning man må/vil gå for 😊